### PR TITLE
Card Design für Bild des Tages

### PR DESCRIPTION
**Pull request by rheaSPK**
- Card Design
- neue Bilder des Tages
- fix für die Animation der Picture Preview

### CODE REVIEW

**Review by olschulz — CHANGES_REQUESTED** [review]
Maybe a bit more margin at the top and bottom and a hover effect similar to the ArchiveCards?

**Review by olschulz — CHANGES_REQUESTED** [review]
Hey, now that I'm able to test it with my local strapi instance I've noticed that

1. You can edit the description in the daily picture as a curator, which probably isn't intended behavior (changes to the description aren't actually getting saved though). This can easily be fixed by setting readonly to true in the editor (like it's done in ArchiveInfo.tsx)
2. A too long line in the description completely breaks the picture
image

**Comment by olschulz** [comment]
We should probably swap out all read-only editor instances for an alternative in the near future. Abusing the jodit editor for read-only text that correctly parses html content is incredibly overkill. the old bp has been using it the same way and it probably isn't as much of a problem if there's just a single text, but e.g. in the comments there's a jodit editor instance for every single comment, because they're editable and therefore might need to render html content. This works for now because we don't have a lot of comments, but could cause lag problems in the future. Using a real read-only alternative would also prevent bugs like forgetting to set it to read-only. @rheaSPK what was the name of the html text parser package you found a while ago again?

**Comment by rheaSPK** [comment]
That's the package for parsing html: https://www.npmjs.com/package/react-html-parser
Yes, I'd be happy to use this instead of Jodit because it makes css editing much more cleaner.

**Review by MariusDoe — CHANGES_REQUESTED** [review]
In addition to my requests, I agree with @olschulz about the editable fields. I think we should look into your suggestion, in combination with something like `dompurify`.

File: projects/bp-gallery/src/components/views/start/DailyPicture.tsx [hunk 1]
```diff
@@ -96,8 +105,17 @@ const DailyPicture = () => {
                 <FolderSpecial /> <a href={pictureArchiveLink}>{pictureArchive} </a>
               </div>
             </div>
-          </div>
-        </>
+            <img
+              className={'w-screen sm:w-auto sm:h-96'}
```
[the remark this answers is not in the document]
Comment (rheaSPK) [reply projects/bp-gallery/src/components/views/start/DailyPicture.tsx:109]: good idea, didn't know that

File: projects/bp-gallery/src/components/views/start/DailyPicture.tsx [hunk 2]
```diff
@@ -73,22 +90,14 @@ const DailyPicture = () => {
...
-              }}
-            />
-            <div className={'flex flex-col max-w-4xl'}>
-              <p className={'line-clamp-5'}>
-                <h4 className={'text-lg my-1'}>{t('pictureFields.descriptions')}:</h4>
+        <div className='flex justify-center'>
+          <Card className='flex flex-col-reverse md:flex-row rounded-md justify-between max-w-4xl max-h-fit'>
+            <div className='p-4 flex flex-col'>
+              <h3 className={'text-2xl'}>{t('common.daily-picture')}</h3>
+              <div className={'line-clamp-5'}>
```
Comment (olschulz) [inline projects/bp-gallery/src/components/views/start/DailyPicture.tsx:97]: maybe more lines? There's a lot of free space with only 5 lines and most descriptions are probably longer.
image

File: projects/bp-gallery/src/components/views/start/DailyPicture.tsx [hunk 3]
```diff
@@ -73,22 +90,14 @@ const DailyPicture = () => {
...
+                <h4 className={'text-lg my-1'}>{t('common.description')}:</h4>
                 <RichText value={description} />
```
Comment (olschulz) [inline projects/bp-gallery/src/components/views/start/DailyPicture.tsx:99]:
```suggestion
                <RichText value={description} className='break-all' />
```
Else too long single lines will break the layout, for example: 
image

**Review by olschulz — CHANGES_REQUESTED** [review]

File: projects/bp-gallery/src/components/views/start/DailyPicture.tsx (hunk 2 above)
Comment (rheaSPK) [reply projects/bp-gallery/src/components/views/start/DailyPicture.tsx:97]: I made it possible to see 10 lines now

**Review by olschulz — APPROVED** [review]
Super :)

**Review by ole1711 — APPROVED** [review]

**Review by MariusDoe — APPROVED** [review]
Let's go :)

**Review by ole1711 — APPROVED** [review]

**Pull request merged**